Add optional 'local' keyword

davidecavestro opened the pull request:
When parsing the output of _ip route get ..._
use an optional _local_ keyword at the beginning of the value in order to avoid failing to parse it for values such as

local 127.0.0.1 dev lo src 127.0.0.1 uid ...

As per issue #2809 filled after a discussion during the weekly help session

## Description

A few sentences describing the overall goals of the pull request's commits.

## Checklist

<!--
  Please review the requirements for each checkbox, and check them
  off (change "[ ]" to "[x]") as you verify that they are complete.
-->

 - [ ] I made sure to update `./CHANGELOG.md`.
 - [ ] I made sure to add any docs changes required for my change (including release notes).
 - [ ] My change is adequately tested.
 - [ ] I updated `DEVELOPING.md` with any any special dev tricks I had to use to work on this code efficiently.
 - [ ] I updated `TELEMETRY.md` if I added, changed, or removed a metric name.
 - [ ] Once my PR is ready to have integration tests ran, I posted the PR in #telepresence-dev in the datawire-oss slack so that the "ok to test" label can be applied.

thallgren commented:
@davidecavestro thanks for your contribution. I approve the change, but could you please sign your commit so that our DCO test pass?

davidecavestro committed:
Add optional 'local' keyword

When parsing the output of _ip route get ..._
use an optional _local_ keyword at the beginning of the value
in order to avoid failing to parse it for values such as

local 127.0.0.1 dev lo src 127.0.0.1 uid ...

As per issue #2809

Signed-off-by: Davide Cavestro <davide.cavestro@gmail.com>

davidecavestro commented:
> could you please sign your commit so that our DCO test pass?

Cool, I re-committed with proper email address and signature

Commit:
Merge branch 'release/v2' into fix-local-routes
